src/App.tsx

```diff
@@ -1,54 +1,124 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
 
+import { Person } from './types/Person';
+
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [apliedQuery, setApliedQuery] = useState('');
+  const [visible, setVisible] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isVisible, setIsVisible] = useState(false);
```